zmon-security-common/src/main/java/de/zalando/zmon/security/tvtoken/TvTokenService.java

```diff
@@ -0,0 +1,41 @@
+package de.zalando.zmon.security.tvtoken;
+
+import org.springframework.util.Base64Utils;
+
+import com.google.common.base.Joiner;
+import com.google.common.base.Splitter;
+
+import de.zalando.zmon.persistence.OnetimeTokensSProcService;
+
+public class TvTokenService {
+
+    private final OnetimeTokensSProcService oneTimeTokenSProcService;
+
+    private final Joiner cookieValueJoiner = Joiner.on("|").useForNull("UNKNOWN");
+    private final Splitter cookieValueSplitter = Splitter.on("|").omitEmptyStrings();
+
+    public TvTokenService(OnetimeTokensSProcService onetimeTokensSProcService) {
+        this.oneTimeTokenSProcService = onetimeTokensSProcService;
+    }
+
+    public String createCookieValue(String token, String bindIp, String sessionId) {
+        String cookieValue = cookieValueJoiner.join(token, bindIp, sessionId);
```

Comment:
Ok, then I will remove it. But what do you think should really be in the cookie-value?